src/cmdexpand.c

```diff
@@ -16,6 +16,9 @@
 static int	cmd_showtail;	// Only show path tail in lists ?
 
 static void	set_expand_context(expand_T *xp);
+static int      ExpandGeneric(expand_T *xp, regmatch_T *regmatch,
+			      int *num_file, char_u ***file,
+			      char_u *((*func)(expand_T *, int)), int escaped);
```

Comment:
We could avoid those 'static' declarations if we re-ordered the
static functions definitions in the order in which they are used.

I did not want to do this as part of this change to avoid
messing up the diff.